keps/sig-network/1860-kube-proxy-IP-node-binding/README.md

```diff
@@ -336,79 +274,36 @@ previous answers based on experience in the field.
 
 ###### Will enabling / using this feature result in any new API calls?
 
-<!--
-Describe them, providing:
-  - API call type (e.g. PATCH pods)
-  - estimated throughput
-  - originating component(s) (e.g. Kubelet, Feature-X-controller)
-Focusing mostly on:
-  - components listing and/or watching resources they didn't before
-  - API calls that may be triggered by changes of some Kubernetes resources
-    (e.g. update of object X triggers new updates of object Y)
-  - periodic API calls to reconcile state (e.g. periodic fetching state,
-    heartbeats, leader election, etc.)
--->
+- API call type - Patch
+- Estimated throughput - 1 per service creation/reconciliation
+- originating component - cloud controller manager / LoadBalancer controller
```

Comment:
The CCM already has to patch the Service with the IP. It will presumably set `ipMode` in the same operation, so there shouldn't be any additional API calls.